Presentation.Web/app/shared/select2Option/select2Option.view.html

```diff
@@ -1,5 +1,6 @@
-﻿<label data-ng-if="label" for="{{ id }}" class="control-label small">{{ label }}</label>
-<input type="hidden" 
+﻿<label data-ng-if="label" for="{{ id }}" class="control-label small">{{ label }} <field-help-text ng-if="isKeyDefined" key="{{helpTextKey}}"></field-help-text></label>
```

Comment:
`{{ someProperty }}` == two way binding = more complexity
`{{::someProperty}}` == one-time rendering

Reply:
no need to use data prefix for 'ng' attributes